frontend/src/metabase/querying/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -29,16 +29,20 @@ export interface FilterColumnPickerProps {
 
 type Section = {
   key?: string;
+  type: string;
   name: string;
   items: (Lib.ColumnMetadata | Lib.SegmentMetadata)[];
   icon?: IconName;
+  alwaysVisible?: boolean;
 };
 
 const CUSTOM_EXPRESSION_SECTION: Section = {
   key: "custom-expression",
+  type: "action",
   name: t`Custom Expression`,
   items: [],
   icon: "filter",
+  alwaysVisible: true,
```

Comment:
I understand that currently `alwaysVisible` will be `true` only when `type` is `"action"`, so this information is equivalent.
Do we plan to allow for different permutations?
If not, WDYT about dropping `alwaysVisible` and assuming sections of type `"action"` are always visible?